components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -902,6 +902,136 @@ public static API getAPI(GovernanceArtifact artifact)
+            api.setFaultSequence(artifact.getAttribute(APIConstants.API_OVERVIEW_FAULTSEQUENCE));
+            api.setDescription(artifact.getAttribute(APIConstants.API_OVERVIEW_DESCRIPTION));
+            api.setResponseCache(artifact.getAttribute(APIConstants.API_OVERVIEW_RESPONSE_CACHING));
+            api.setType(artifact.getAttribute(APIConstants.API_OVERVIEW_TYPE));
+            int cacheTimeout = APIConstants.API_RESPONSE_CACHE_TIMEOUT;
+            try {
+                cacheTimeout = Integer.parseInt(artifact.getAttribute(APIConstants.API_OVERVIEW_CACHE_TIMEOUT));
+            } catch (NumberFormatException e) {
+                //ignore
+            }
+            api.setCacheTimeout(cacheTimeout);
+
+            boolean isGlobalThrottlingEnabled = APIUtil.isAdvanceThrottlingEnabled();
+
+            if (isGlobalThrottlingEnabled) {
+                String apiLevelTier = ApiMgtDAO.getInstance().getAPILevelTier(apiId);
+                api.setApiLevelPolicy(apiLevelTier);
+
+                Set<Tier> availablePolicy = new HashSet<Tier>();
+                String[] subscriptionPolicy = ApiMgtDAO.getInstance().getPolicyNames(PolicyConstants.POLICY_LEVEL_SUB,
+                        replaceEmailDomainBack(providerName));
+                List<String> definedPolicyNames = Arrays.asList(subscriptionPolicy);
+                String policies = artifact.getAttribute(APIConstants.API_OVERVIEW_TIER);
+                if (policies != null && !"".equals(policies)) {
+                    String[] policyNames = policies.split("\\|\\|");
+                    for (String policyName : policyNames) {
+                        if (definedPolicyNames.contains(policyName) || APIConstants.UNLIMITED_TIER.equals(policyName)) {
+                            Tier p = new Tier(policyName);
+                            availablePolicy.add(p);
+                        } else {
+                            log.warn("Unknown policy: " + policyName + " found on API: " + apiName);
+                        }
+                    }
+                }
+
+                api.addAvailableTiers(availablePolicy);
+                String tenantDomainName = MultitenantUtils.getTenantDomain(replaceEmailDomainBack(providerName));
+                api.setMonetizationCategory(getAPIMonetizationCategory(availablePolicy, tenantDomainName));
+            } else {
+                //deprecated throttling method
+                Set<Tier> availableTier = new HashSet<Tier>();
+                String tiers = artifact.getAttribute(APIConstants.API_OVERVIEW_TIER);
+                String tenantDomainName = MultitenantUtils.getTenantDomain(replaceEmailDomainBack(providerName));
+                if (tiers != null) {
+                    String[] tierNames = tiers.split("\\|\\|");
+                    for (String tierName : tierNames) {
+                        Tier tier = new Tier(tierName);
+                        availableTier.add(tier);
+
```

Comment:
Please remove the unwanted newlines.

Reply:
https://github.com/wso2/carbon-apimgt/pull/7953/commits/b2d0a6f35201ec88c8880a1dad6a00e4cfb5e9bf